src/main/resources/templates/js/trends-chart.js.vm

```diff
@@ -1,20 +1,23 @@
+#macro(stringArray $array) [#foreach($element in $array) "$element", #end] #end
+#macro(numberArray $array) [#foreach($element in $array) $element, #end] #end
+
 $(document).ready(function() {
 
-    var buildNumbers = $buildNumbers;
+    var buildNumbers = #stringArray($buildNumbers);
 
-    var failedFeatures = $failedFeatures;
-    var totalFeatures = $totalFeatures;
+    var failedFeatures = #numberArray($failedFeatures);
+    var totalFeatures = #numberArray($totalFeatures);
 
-    var failedScenarios = $failedScenarios;
-    var totalScenarios = $totalScenarios;
+    var failedScenarios = #numberArray($failedScenarios);
+    var totalScenarios = #numberArray($totalScenarios);
 
-    var passedSteps = $passedSteps;
-    var failedSteps = $failedSteps;
-    var skippedSteps = $skippedSteps;
-    var pendingSteps = $pendingSteps;
-    var undefinedSteps = $undefinedSteps;
+    var passedSteps = #numberArray($passedSteps);
+    var failedSteps = #numberArray($failedSteps);
+    var skippedSteps = #numberArray($skippedSteps);
+    var pendingSteps = #numberArray($pendingSteps);
+    var undefinedSteps = #numberArray($undefinedSteps);
 
-    var durations = $durations;
+    var durations = #numberArray($durations);
 
```

Comment:
number array and string array are not strictly needed, javascript will coerce  strings that represent numbers into numbers when needed but this just feels better, more precise.